src/explore-education-statistics-common/src/components/AccordionSection.tsx

```diff
@@ -58,78 +59,82 @@ const AccordionSection = ({
   const contentId = `${id}-content`;
   const anchorId = `${anchorLinkIdPrefix}-${kebabCase(heading)}`;
 
   return (
-    <div
-      className={classNames(classes.section, styles.section, className, {
-        [classes.expanded]: open,
-      })}
-      data-testid="accordionSection"
-      id={id}
-    >
-      <div className="govuk-accordion__section-header">
-        {anchorLinkUrl && (
-          <CopyLinkButton
-            className={styles.copyLinkButton}
-            url={anchorLinkUrl(anchorId)}
-          />
-        )}
-        {header ??
-          createElement(
-            headingTag,
-            {
-              className: classes.sectionHeading,
-              id: anchorId,
-            },
-            isMounted ? (
-              <button
-                aria-controls={contentId}
-                aria-expanded={open}
-                className={classes.sectionButton}
-                id={headingId}
-                type="button"
-                onClick={() => {
-                  if (onToggle) {
-                    onToggle(!open, id);
-                  }
-                }}
-              >
-                {heading}
-                <span aria-hidden className="govuk-accordion__icon" />
-              </button>
-            ) : (
-              <span id={headingId} className={classes.sectionButton}>
-                {heading}
-              </span>
-            ),
-          )}
-
-        {caption && (
-          <div className="govuk-accordion__section-summary govuk-body">
-            {caption}
-          </div>
-        )}
-      </div>
+    <>
       <div
-        aria-labelledby={headingId}
-        className={classNames(
-          classes.sectionContent,
-          onMounted({ [styles.sectionContentCollapsed]: !open }),
-        )}
-        id={contentId}
+        className={classNames(classes.section, styles.section, className, {
+          [classes.expanded]: open,
+          [styles.hasAnchorLink]: anchorLinkUrl,
+        })}
+        data-testid="accordionSection"
+        id={id}
       >
-        {typeof children === 'function'
-          ? children({ open, contentId, headingId })
-          : children}
+        <div className="govuk-accordion__section-header">
+          {anchorLinkUrl && (
+            <CopyLinkButton
+              className={styles.copyLinkButton}
+              url={anchorLinkUrl(anchorId)}
+            />
+          )}
+          {header ??
+            createElement(
+              headingTag,
+              {
+                className: classes.sectionHeading,
+                id: anchorId,
+              },
+              isMounted ? (
+                <button
+                  aria-controls={contentId}
+                  aria-expanded={open}
+                  className={classes.sectionButton}
+                  id={headingId}
+                  type="button"
+                  onClick={() => {
+                    if (onToggle) {
+                      onToggle(!open, id);
```

Comment:
nitpick: I believe we can simplify this to `onClick={() => { onToggle?.(!open, id) }}`